src/config.ts

```diff
@@ -9,8 +9,6 @@ function readConfig()
         requiredEnvs = [
             'NIGHTSCOUT_API_TOKEN',
             'NIGHTSCOUT_URL',
-            'LINK_UP_USERNAME',
```

Comment:
Rather than removing USERNAME and PASSWORD from the required args, could you implement a check that validates that either USERNAME and PASSWORD or a CSV is provided?